panel/lxqtpanel.cpp

```diff
@@ -374,7 +372,7 @@ int LXQtPanel::getReserveDimension()
     return mHidable ? PANEL_HIDE_SIZE : qMax(PANEL_MINIMUM_SIZE, mPanelSize);
 }
 
-void LXQtPanel::setPanelGeometry(bool animate)
+void LXQtPanel::setPanelGeometry(bool animate, bool firstTime)
```

Comment:
> I don't get this firstTime flag.

Without it, there would be a mess. Since one test is worth a million words, try it ;) In short, two things would happen at start-up: (1) A long-distance animation would be shown, and (2) the available desktop geometry would be not only wrong but ridiculous.